frontend/src/components/calendar/BigCalendar/BigCalendar.tsx

```diff
@@ -1,33 +1,75 @@
+import { useEffect, useState } from 'react';
 import { Calendar, CalendarProps, momentLocalizer } from 'react-big-calendar';
+import { toast } from 'react-toastify';
 import moment from 'moment';
+import { DateTime } from 'ts-luxon';
+import withDragAndDrop, {
+  withDragAndDropProps,
+} from 'react-big-calendar/lib/addons/dragAndDrop';
+import 'react-big-calendar/lib/addons/dragAndDrop/styles.css';
+
+import useFetch from '../../../hooks/useFetch';
+import LoadingScreen from '../../ui/Loading/LoadingScreen';
+import EventCard from '../../ui/EventCard/EventCard';
 
 import './Calendar.scss';
 
-const BigCalendar = (props: Omit<CalendarProps, 'localizer'>) => {
+const DnDCalendar = withDragAndDrop(Calendar);
+type DnDType = CalendarProps & withDragAndDropProps;
+type CustomCalendarProps = Omit<DnDType, 'components' | 'localizer'>;
+
+const BigCalendar = (props: CustomCalendarProps) => {
   const localizer = momentLocalizer(moment);
+  const [events, setEvents] = useState([]);
+  const { fetchData: getEvents } = useFetch('GET', ['events']);
+  const [loading, setLoading] = useState(true);
```

Comment:
remove the loading here and put it in line 24

Reply:
const { fetchData: getEvents, loading: isLoading } = useFetch('GET', ['events']);